ldi-orchestrator/ldio-connectors/ldio-kakfa-in/README.MD

```diff
@@ -0,0 +1,63 @@
+# Kafka OUT
+
+This output component listens to messages from a [kafka topic](https://kafka.apache.org).
+
+Two security protocols are supported:
+- [NO SECURITY](#no-security)
+- [SASL SSL PLAIN](#sasl-ssl-plain)
+
+## NO SECURITY
+
+The following configuration is supported:
+
+```json
+outputs:
+  - name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioKafkaOut
+    config:
+      content-type: { any content type supported by Apache Jena }
+      topics: { comma separated list of topics }
+      bootstrap-servers: { uri of the bootstrap server }
+      group-id: { optional group identifier }
```

Comment:
Maybe to mention that if not defined, a random one will be generated. ldio-[serverName]-[pipelineName]